rpc/backend/node_info.go

```diff
@@ -345,7 +346,7 @@ func (b *Backend) RPCMinGasPrice() int64 {
 	}
 
 	minGasPrice := b.cfg.GetMinGasPrices()
-	amt := minGasPrice.AmountOf(evmParams.Params.EvmDenom).TruncateInt64()
+	amt := minGasPrice.AmountOf(evmParams.Params.GasDenom).TruncateInt64()
```

Comment:
look like every sequencer need to setting min-gas-prices for both denom (native + ibc)

Reply:
they need to set min-gas-prices only for the denom specified in `GasDenom`. by default it equals native, so nothing should change. but yeah, this needs to be adjusted in case `GasDenom` changes.